src/data.py

```diff
@@ -306,7 +330,7 @@ def __getitem__(self, index: int) -> Union[TrainBatch, TestBatch]:
         crop = self.crops[index]
         if self.mask_fps is not None:
             mask_t = Image.open(self._dataset_folder / self.mask_fps[index])
-            mask = self._target_transform(mask_t)
+            mask = self._target_transform(mask_t).squeeze(0)
```

Comment:
```suggestion
            mask = self._target_transform(mask_t)
```
I'm not sure this if this is needed....

Reply:
Well, it runs without it, so 🤷‍♂️